### PR TITLE
Diagnostics 01-20-2016

### PR DESCRIPTION
**Pull request by kenvng**
Diagnostics 01-20-2016

### CODE REVIEW

**Comment by kenvng** [comment]
I noticed that above a comment saying "This branch has conflicts that must be resolved".  Just curious if I did something wrong?

File: diagnostic.md [hunk 1]
```diff
@@ -7,27 +7,36 @@ Write your answers inside this file, where it's indicated by the comments.
...
+    -->
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    cd into the folder/directory with the file you need to work on and
+    enter: atom <filename with extension> to open it in atom.
```
Comment (GA-MEB) [inline diagnostic.md:20]: At least in the context of this class's diagnostics, before you open your editor and begin editing, you should create a new branch and switch to it.

File: diagnostic.md [hunk 2]
```diff
@@ -7,27 +7,36 @@ Write your answers inside this file, where it's indicated by the comments.
...
+     -->
     ```
 
 3.  What command do we use to create a new branch? Name this branch `response`
     in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    git branch <name>
```
Comment (GA-MEB) [inline diagnostic.md:29]: Careful with the directions - we were looking specifically for `git branch response` (or, alternatively, `git checkout -b response`)

File: diagnostic.md [hunk 3]
```diff
@@ -35,11 +44,16 @@ Write your answers inside this file, where it's indicated by the comments.
     directory and named `diagnostic.md`.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    <!--
+    git add ., but recommended not to do that by instructor(s)
```
Comment (GA-MEB) [inline diagnostic.md:48]: That's right, please do not do `git add .` if you can help it. You might accidentally sweep things into your commits that shouldn't be there.

File: diagnostic.md [hunk 4]
```diff
@@ -35,11 +44,16 @@ Write your answers inside this file, where it's indicated by the comments.
...
+    git add dianostic.md, is recommended
```
Comment (GA-MEB) [inline diagnostic.md:49]: Correct.

**Comment by GA-MEB** [comment]
5/6 (-1 on Q2)

**Pull request closed without merging**